numpy/lib/tests/test_histograms.py

```diff
@@ -249,6 +249,10 @@ def test_unsigned_monotonicity_check(self):
         with assert_raises(ValueError):
             hist, edges = np.histogram(arr, bins=bins)
 
+        # gh-12107
+        # Shouldn't raise an error
+        hist, edges = np.histogram([True, False])
```

Comment:
You should probably also test with `np.bool_`, which is the numpy boolean type, different from the python version. You can get the type with `np.bool_(1)`

Reply:
@charris good point, added.